demos/twa-location-delegation/src/main/java/com/google/browser/examples/twa_location_delegation/LocationDelegationService.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.browser.examples.twa_location_delegation;
+
+import com.google.androidbrowserhelper.trusted.DelegationService;
+import com.google.androidbrowserhelper.locationdelegation.LocationDelegationExtraCommandHandler;
+
+public class LocationDelegationService extends DelegationService {
```

Comment:
I'm not sure this should be called "LocationDelegationService" - yes, what this does is location delegation, but the way we've designed the API is that multiple different features can go in here. Eg, if someone wanted to add the Payments Service in as well, they'd just add a line in the constructor.

Maybe call this ExtraFeaturesService or OptionalFeaturesService or something? (Open to better name suggestions.)

Reply:
Renamed to ExtraFeaturesService